resources/clj-kondo.exports/com.gfredericks/test.chuck/config.edn

```diff
@@ -0,0 +1,4 @@
+{:hooks
+ {:analyze-call
+  {com.gfredericks.test.chuck.clojure-test/checking
+   hooks.checking/checking}}}
```

Comment:
is it idiomatic with kondo to such a generic namespace for the validation code? E.g., if some other library also had something called `checking`, could this be a problem?

Reply:
It's unclear to me. Some projects use something more specific, like https://github.com/CrypticButter/snoop/blob/main/resources/clj-kondo.exports/com.crypticbutter/snoop/clj_kondo/crypticbutter/snoop.clj

Reply:
Others are simpler https://github.com/liquidz/merr/blob/main/resources/clj-kondo.exports/merr/merr/hooks/merr.clj 

However `merr` is a less common name than `checking`

Reply:
if there's not a predominate Normal Way Of Doing Things, I think I'd prefer a specific namespace, that at least contains `com.gfredericks.test.chuck`

otherwise this looks good, thanks!

Reply:
Sounds good. I'll play around with that later. Thanks Gary

Reply:
Updated